lib/motions/move-to-definition-motion.coffee

```diff
@@ -0,0 +1,40 @@
+_ = require 'underscore-plus'
+{Input} = require '../view-models/view-model'
+{SearchCurrentWord} = require './search-motion'
+
+module.exports =
+class MoveToDefinition extends SearchCurrentWord
+  constructor: (@editor, @vimState) ->
+    super(@editor, @vimState)
+
+    # FIXME: This must depend on the current language
+    # NOTE: The only modification here is the addition of the $
+    defaultIsKeyword = "[@$a-zA-Z0-9_\-]+"
+    userIsKeyword = atom.config.get('vim-mode.iskeyword')
+    @keywordRegex = new RegExp(userIsKeyword or defaultIsKeyword)
+
+    searchString = @getCurrentWordMatch()
+    @input = new Input(searchString)
```

Comment:
`$` is added in the default keyword regex, but if the user has `iskeyword` set, theirs will be the same as for `SearchCurrentWord` then. I'd suggest that, until this regex depends on the language, we just rely on `iskeyword` for the right behaviour. 

If you agree, this constructor could go.